solver/src/driver.rs

```diff
@@ -248,8 +248,8 @@ impl Driver {
                 Err(err) => err,
             };
             tracing::error!(
-                "settlement simulation failed right before submission AND for block {} which was current when liquidity was fetched:\n{:?}",
-                current_block_during_liquidity_fetch, error_at_earlier_block
+                "settlement simulation for {} solver failed right before submission AND for block {} which was current when liquidity was fetched:\n{:?}",
```

Comment:
Because the tenderly link is very long, and we sometimes run into maximum message size issues, can we shorten this message to save a few characters? For example:
```suggestion
                "{} settlement simulation failed at submission and block {}:\n{:?}",
```